content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -19,18 +19,96 @@ Please contact R3 if you are interested in joining the Beta program.
 
 This section describes the new features in Corda 5.0 Beta 2.0.
 
+### Cluster Configuration
 
+#### Mutual TLS
+It is now possible to configure Corda cluster gateways to connect with each other via [mutual TLS]({{< relref "../operating/mutual-tls-connections.md" >}}).
+
+#### Messaging Maximum Size
+A new `messaging` [configuration field]({{< relref "../operating/configuration/config-overview.md" >}}), `maxAllowedMessageSize`, enables you to specify the maximum size of a message sent from a Corda worker to the message bus. Corda breaks messages that exceed this size into smaller messages before sending.
+The `maxAllowedMessageSize` value must be lower than the maximum message size configured on the message bus itself, for example, Kafka. If the Corda configuration value is set to a value higher than that of the message bus, the messages will not send.
+
+#### Registration Approval
+A network operator can now configure a membership group to require that the operator must manually approve (or decline) member registration requests. Requests satisfying specific criteria require manual approval, while others are approved automatically. The operator can also pre-authenticate specific members, allowing them to bypass the standard approval rules defined for the group. The operator can further configure pre-authentication to specify that certain changes to the member’s context must be manually reviewed. For more information, see [Manual Registration Approval]({{< relref "../operating/registration-approval.md" >}}).
+
+### CorDapp Development
+
+#### Java API
+The Kotlin API has been replaced with a Java API. Kotlin and Java developers can now use this to build CorDapps.
+
+#### Sign Without Arguments
+`UtxoTransactionBuilder.toSignedTransaction(signatory: PublicKey)` and `ConsensualTransactionBuilder.toSignedTransaction(signatory: PublicKey)` have been removed. New versions without arguments are now available. These sign with all the available required keys.
+
+#### Transaction Notary
+`UtxoLedgerTx.notary` is now available. This is the notary used for notarising this transaction.
+
+#### UtxoLedgerService
+`UtxoLedgerService.resolve()` functions have been added to the API.
+
+#### Crypto API
+`SignatureSpec` is now an explicit field in `DigitalSignatureMetadata`. In previous versions, it was necessary to add it in the `DigitalSignatureMetadata.properties` map.
+
+#### P2P Avro Types
+Avro types for the peer-to-peer communications layer were moved from the `net.corda.p2p` namespace to the `net.corda.data.p2p` namespace.
+
+#### Find Signing Keys
+A new function, `findMySigningKeys`, has been added to the `SigningService` interface. This function checks if the specified set of keys are owned by the caller. It returns a mapping from the requested key to the same key if it is owned by the caller or to null if the key is not owned by the caller.
+In the case of a composite key, it maps the composite key to the firstly found composite key leaf.
```

Comment:
I understand this is being taken from `findMySigningKeys` JavaDoc. We have updated it recently with [#815](https://github.com/corda/corda-api/pull/815). Not really sure if there is anything to take out of its updated version and add here though.

Reply:
@kyriathar I made an update if you could re-review, please?